scss/product/components/_forms.scss

```diff
@@ -197,87 +180,97 @@ select.form-control-lg {
   margin-right: -5px;
   margin-left: -5px;
 
-  > .col,
-  > [class*="col-"] {
+  > [class*="col-"],
+  > .col {
     padding-right: 5px;
     padding-left: 5px;
   }
 }
-
 //counter input fields
 //
 
 .counter-input {
   position: relative;
   display: inline-flex;
-  justify-content:center;
+  justify-content: center;
   vertical-align: middle; // match .btn alignment given font-size hack above
   border: 2px solid $gray-200;
   border-radius: 4px;
   width: 200px;
+
   > .btn {
     position: relative;
     flex: 1 1 auto;
+    background: $white;
     color: $gray-200;
     // Bring the hover, focused, and "active" buttons to the front to overlay
     // the borders properly
     @include hover {
+      background: $gray-000;
       z-index: 1;
+      color: $gray-300;
     }
-    &:focus,
+
+    &.active,
     &:active,
-    &.active {
+    &:focus {
       z-index: 1;
     }
   }
   // Prevent double borders when buttons are next to each other
-  > .btn:not(:first-child),
-  > .btn-group:not(:first-child) {
+  > .btn-group:not(:first-child),
+  > .btn:not(:first-child) {
     margin-left: -$btn-border-width;
   }
-
   // Reset rounded corners
-  > .btn:not(:last-child):not(.dropdown-toggle),
-  > .btn-group:not(:last-child) > .btn {
+  > .btn-group:not(:last-child) > .btn,
+  > .btn:not(:last-child):not(.dropdown-toggle) {
     @include border-right-radius(0);
   }
 
-> .form-control {
-  box-shadow: none;
-  text-align: center;
-  @include border-right-radius(0);
-  @include border-left-radius(0);
-}
-
-> .counter-input-select {
-  border-left: 2px solid $gray-200;
-  -webkit-appearance: none;
-  -moz-appearance: none;
-}
+  > .btn-group:not(:first-child) > .btn,
+  > .btn:not(:first-child) {
+    @include border-left-radius(0);
+  }
 
-  > .btn:not(:first-child),
-  > .btn-group:not(:first-child) > .btn {
+  .form-control {
+    box-shadow: none;
+    text-align: center;
+    @include border-right-radius(0);
     @include border-left-radius(0);
   }
+
+  .counter-input-number-field { }
```

Comment:
Since no CSS is added for this class, this line `.counter-input-number-field { }` can be removed

Reply:
removed